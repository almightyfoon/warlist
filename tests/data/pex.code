c8501b_-0p5C4Qcncng4g45g5gc95f5f___-0WgM4B4B4M4M5H5H5H5H5H5H5H5A5g5g5f
