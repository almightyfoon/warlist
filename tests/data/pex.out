https://conflictchamber.com/#c8501b_-0p5C4Qcncng4g45g5gmV5f5f___-0WgM4B4B4M4M5H5H5H5H5H5H5H5A5g5g5f

Circle Army - 75 / 75 points
[Theme] The Bones of Orboros

[Baldur 2] Baldur the Stonesoul [+30]
 - Megalith [20]
 - Woldwrath [37]
 - Woldwrath [37]
Blackclad Stoneshaper [0(3)]
Blackclad Stoneshaper [3]
Blackclad Wayfarer [0(4)]
Blackclad Wayfarer [0(4)]
Gallows Grove [2]
Shifting Stones [3]
Shifting Stones [3]

Circle Army - 75 / 75 points
[Theme] Call of the Wild

[Una 2] Una the Skyhunter [+29]
 - Gorax Rager [7]
 - Gorax Rager [7]
 - Pureblood Warpwolf [17]
 - Pureblood Warpwolf [17]
 - Scarsfell Griffon [8]
 - Scarsfell Griffon [8]
 - Scarsfell Griffon [8]
 - Scarsfell Griffon [8]
 - Scarsfell Griffon [8]
 - Scarsfell Griffon [8]
 - Scarsfell Griffon [8]
 - Druid Wilder [0(4)]
Blackclad Wayfarer [0(4)]
Blackclad Wayfarer [0(4)]
Shifting Stones [0(3)]
